app/helpers/spotlight/search_configurations_helper.rb

```diff
@@ -11,9 +11,9 @@ def translate_sort_fields(sort_config)
       sort_description ||= sort_config[:sort].split(',').map do |sort|
         sort_field, sort_order = sort.split(' ')
         safe_join([
-          t(:"spotlight.search_configurations.sort.keys.#{sort_field.strip}", default: sort_field.humanize.downcase),
-          t(:"spotlight.search_configurations.sort.keys.#{sort_order.strip}", default: '')
-        ], ' ')
+                    t(:"spotlight.search_configurations.sort.keys.#{sort_field.strip}", default: sort_field.humanize.downcase),
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.